ui/react-components/components/ListView/ListView.module.scss

```diff
@@ -0,0 +1,79 @@
+@import '../../variables.scss';
+@import url('https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css');
```

Comment:
is it possible to just load font in storybook? Instead of the main component style?

Reply:
Added the font-awesome react package, without that, the svgs would not rerender, hence. 

With the SVG and JS way, the JS to parse our HTML and add the SVG may fire before React has time to mount it's components. So we have to find a way to parse the HTML once React has mounted its components.

The package solves that issue. 

Also added selection of a particular appointment. Deselecting an appointment by clicking elsewhere, I think could be done as part of integration